### ISSUE TITLE
/dev/serial/by-id missing

### ISSUE THREAD

**Issue by AshtonKem** [NixOS/nixpkgs#223258]
### Describe the bug
The symlinks in /dev/serial/by-id are missing, which makes using USB devices sensitive to the order that they were inserted.

### Steps To Reproduce
1. Insert USB serial device 

### Expected behavior
A symlink to /dev/serial/by-id, which allows for stable access to the same device regardless of insertion order.

### Additional context
I'm using a USB Zigbee coordinator, which is currently only mounted at /dev/ttyUSB0.

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[root@r420:/home/ashton]# nix-info -m
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.85, NixOS, 22.05 (Quokka), 22.05.4690.08741686397`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.1`
 - channels(root): `"nixos-22.05"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by IndeedNotJames** [comment]
Can't reproduce it with mine.


> A symlink to /dev/serial/by-id, which allows for stable access to the same device regardless of insertion order.

I might have misunderstood you here, but the expected behavior is `/dev/serial/by-id/<id>` pointing to `../../ttyUSB0` or whatever.
Essentially `ln -s /dev/serial/by-id/<id> ../../ttyUSB0`
Not the other way around.

Also, consider updating to 22.11, as 22.05 is EOL.

**Comment by AshtonKem** [comment]
You're correct, I was slightly sloppy in my description. I expect a symlink from '/dev/serial/by-id/<foo>' to '/dev/ttyUSBX', but no such links exist. 

I will attempt updating and see if that resolves the issue.

**Comment by AshtonKem** [comment]
Seems fixed in 22.11. So I guess this _was_ a bug, but is now fixed. 

Thank you for the suggestion.
